index.js

```diff
@@ -56,7 +56,8 @@ module.exports = {
 				'unicorn/prefer-node-append': 'error',
 				'unicorn/prefer-query-selector': 'error',
 				'unicorn/prefer-node-remove': 'error',
-				'unicorn/prefer-text-content': 'error'
+				'unicorn/prefer-text-content': 'error',
+				'unicorn/prefer-key-over-key-code': 'error'
```

Comment:
@sindresorhus Should this rule be named `prefer-key` or `prefer-event-key`, without the "over something else" part, like the other rules, or is the current name fine?

Reply:
Yes, it should be `prefer-event-key`.